system/data/es/systems/condensed.json

```diff
@@ -0,0 +1,284 @@
+                        }
+                    },
+                    {
+                        "name": "Saber",
+                        "type": "skill",
+                        "data": {
+                            "description": "Conocimiento especializado o arcano fuera del alcance de la habilidad de Conocimiento."
+                        }
+                    },
+                    {
+                        "name": "Percepción",
+                        "type": "skill",
+                        "data": {
+                            "description": "Habilidad para dar cuenta de detalles, presentir un problema o simplemente ser mas perceptivo. "
+                        }
+                    },
+                    {
+                        "name": "Físico",
+                        "type": "skill",
+                        "data": {
+                            "description": "Fuerza bruta y resistencia"
+                        }
+                    },
+                    {
+                        "name": "Provocar",
+                        "type": "skill",
+                        "data": {
+                            "description": "Capacidad para incitar a otros para que actuen según tu voluntad a través de la manipulación."
+                        }
+                    },
+                    {
+                        "name": "Carisma",
+                        "type": "skill",
+                        "data": {
+                            "description": "Crear vínculos con otros y trabajar de manera conjunta."
+                        }
+                    },
+                    {
+                        "name": "Recursos",
+                        "type": "skill",
+                        "data": {
+                            "description": "Acceso a recursos materiales (no solo dinero)."
+                        }
+                    },
+                    {
+                        "name": "Disparar",
+                        "type": "skill",
+                        "data": {
+                            "description": "Todo tipo de ataques a distancia, desde armas de fuego, lanzar cuchillos o arco y flechas."
+                        }
+                    },
+                    {
+                        "name": "Sigilo",
+                        "type": "skill",
+                        "data": {
+                            "description": "Evitar ser detectado y esconderte cuando es necesario."
+                        }
+                    },
+                    {
+                        "name": "Voluntad",
+                        "type": "skill",
+                        "data": {
+                            "description": "Fortaleza mental, la capacidad de evitar la tentación o soportar traumas."
+                        }
+                    }
+                ]
+            }
+        ],
+        "Tracks": [
+            {
+                "label": "Estilo Core (Básico)",
```

Comment:
See my comment in line 2 of core.json: I'd use only "Básico" to refer to Core everywhere.